ai_review/algo/utils.py

```
@@ -0,0 +1,1248 @@
+    # first fallback - try to convert 'relevant line: ...' to relevant line: |-\n        ...'
+    response_text_lines_copy = response_text_lines.copy()
+    for i in range(0, len(response_text_lines_copy)):
+        for key in keys_yaml:
+            if key in response_text_lines_copy[i] and not '|' in response_text_lines_copy[i]:
+                response_text_lines_copy[i] = response_text_lines_copy[i].replace(f'{key}',
+                                                                                  f'{key} |\n        ')
+    try:
+        data = yaml.safe_load('\n'.join(response_text_lines_copy))
+        get_logger().info(f"Successfully parsed AI prediction after adding |-\n")
+        return data
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)